src/managers/RoleManager.js

```diff
@@ -132,6 +132,24 @@ class RoleManager extends BaseManager {
     return this.cache.get(this.guild.id);
   }
 
+  /**
+   * The premium subscriber role of the guild, if any
+   * @type {?Role}
+   * @readonly
+   */
+  get premiumSubscriberRole() {
+    return this.cache.find(role => role.premiumSubscriberRole);
+  }
```

Comment:
should this return null if the role isn't found? or should it return undefined still

Reply:
in my opinion getters should propagate the find value in that case (which returns undefined in v12). Which would be consistent with other similar API behavior (<https://discord.js.org/#/docs/main/stable/class/User?scrollTo=avatarURL>, to name one)
I'll let our core devs decide on that one